tests/gcloud/test_GCloudStorage.py

```diff
@@ -79,9 +90,52 @@ def test_incorrect_data_type_to_upload(self, mock_gcloud):
                                     dt,
                                     file_path,
                                     file_name)
-    
+
+    @mock.patch('SwissKnife.gcloud.GCloudStorage.gcloud')
+    def test_valid_path_in_list_blobs_with_prefix(self, mock_gcloud):
+
```

Comment:
You know I love to say this...

`spacing`

Reply:
Agree

Reply:
![image](https://user-images.githubusercontent.com/564716/83243669-89619580-a19e-11ea-94a5-7251cbc92fa5.png)